src/client/autoVersion.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var ping = require('../ping');
+var debug = require('../debug');
+var states = require('../states');
+var assert = require('assert');
+var minecraft_data = require('minecraft-data');
+
+// Get the minecraft-data version string for a protocol version
+// TODO: add to node-minecraft-data index (protocol to newest release, if multiple)
+function protocolVersion2MinecraftVersion(n) {
```

Comment:
this can now be implemented using https://github.com/PrismarineJS/node-minecraft-data/blob/master/example.js#L19 I think